os/fs/dirent/fs_opendir.c

```diff
@@ -246,9 +246,13 @@ FAR DIR *opendir(FAR const char *path)
 	/* If we are given 'nothing' then we will interpret this as
 	 * request for the root inode.
 	 */
+	if (!path || *path == 0) {
```

Comment:
previous is right, as I know it means root.
Is there any problem because of this condition?

Reply:
Also *path == '\0' is correct, I think